lib/Migration/Version001000Date20240712000000.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/** @noinspection PhpUnused */
+
+declare(strict_types=1);
+
+namespace OCA\Tables\Migration;
+
+use Closure;
+use OCP\DB\Exception;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\Types;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version001000Date20240712000000 extends SimpleMigrationStep {
```

Comment:
```suggestion
class Version000800Date20240712000000 extends SimpleMigrationStep {
```

and also name the filename like that The version there is following the app version where we currently target 0.8